core/src/main/java/com/radixdlt/environment/rx/ObservableProvider.java

```diff
@@ -81,7 +82,7 @@ public final class ObservableProvider<T> implements Provider<Observable<T>> {
     this.t = null;
   }
 
-  ObservableProvider(TypeLiteral<T> t) {
+  ObservableProvider(TypeLiteral<T> t) { // TODO: SIY: unused
```

Comment:
if so, it's an excellent opportunity to remove it

Reply:
Good point. Removed.